src/ocean/text/convert/Formatter.d

```diff
@@ -13,6 +13,52 @@
     - `snformat` is equivalent to `std.format.sformat`
     - `sformat` is roughly equivalent to `std.format.formattedWrite`
 
+    Format_specifier:
+
+    The Formatter uses a format specification similar to C#/.NET over
+    the traditional `printf` style.
+    As a result, the most simple usage is to call:
+    ---
+    format("This value will be default formatted: {}", value);
+    ---
+
+    More specific formatting options are however available.
+
+    The format specifier is defined as follows:
+
+    '{'[INDEX][WIDTH_CHAR[ALIGN_LEFT_CHAR][ALIGN_WIDTH][\s*]][':'][FORMAT_STRING]'}'
```

Comment:
It's not clear to me what `[\s*]` means. Some kind of regex, I presume, but we might want to make this doc readable to people who don't have such knowledge.

Reply:
Why `[:]` but `[ALIGN_LEFT_CHAR]` (not `[-]`)?

Reply:
Yeah `\s` is the whitespace class. However that's slightly wrong since only spaces are skipped. will correct with `[' '*]`, how does it sound ?

Reply:
Because `ALIGN_LEFT_CHAR` has an impact on the output (thus I refer to it later) but ':' doesn't

Reply:
Yeah `[' '*]` is a bit more intuitively obvious.